src/dotnet-proj-info/Program.fs

```diff
@@ -111,12 +129,21 @@ let realMain argv = attempt {
         | _ -> Error (InvalidArgsState "specify only one get argument")
 
     let exec getArgs additionalArgs = attempt {
-        let msbuildExec = dotnetMsbuild (runCmd log)
+        let msbuildExec =
+            if isDotnetSdk then
+                dotnetMsbuild (runCmd log)
+            else
+                msbuild (MSBuildExePath.Path "msbuild") (runCmd log)
```

Comment:
add `--msbuild` argument to specify the msbuild path (or xbuild), but default to "msbuild"